### DISCUSSION TITLE
Good afternoon, why in the free plan I can not upload a video that weighs 714 mb??

### DISCUSSION THREAD

**Discussion by Maksim22886** [supabase/supabase#22747] [Questions]
<img width="399" alt="image" src="https://github.com/supabase/supabase/assets/120506295/49fcf03f-83b0-40a5-8889-417ae5ad5830">

**Comment by GaryAustin1** [comment]
![image](https://github.com/supabase/supabase/assets/54564956/7fb14c6e-8ed5-439c-be0c-5a04618b6134)
The limit is 50MB

**Reply by Maksim22886** [reply]
If I buy a PRO what is my limit?

**Reply by GaryAustin1** [reply]
https://supabase.com/pricing
You should look there for these type of questions.
It says 50GB in the next section after the Free one I showed.

But you should also read here on methods for uploading large files:
https://supabase.com/docs/guides/storage/uploads/standard-uploads
